### ISSUE TITLE
DBW: single-use event listeners instead of add/remove patterns

### ISSUE THREAD

**Issue by ebidel** [GoogleChrome/lighthouse#872]
https://www.chromestatus.com/feature/5630331130478592

```
for all nodes
  gather event listeners

for each event listener, check if it is calling removeEventListener instead of itself (with the same params as it was created with it
```

Suggest `addEventListener('event', ... {once: true})` instead!

**Comment by ebidel** [comment]
@paulirish for this we need the `once` flag to be added to https://chromedevtools.github.io/debugger-protocol-viewer/tot/DOMDebugger/#type-EventListener. Similar to how there's a `passive` flag on the listener.

Who do we talk to?

**Comment by paulirish** [comment]
dtapuska

**Comment by ebidel** [comment]
This was recently added to chrome: https://codereview.chromium.org/2490303003

**Comment by paulirish** [comment]
closed via #937
